package.json

```diff
@@ -206,7 +206,7 @@
     "swagger-client": "^2.1.32",
     "swagger-js-codegen": "git+https://github.com/cBioPortal/swagger-js-codegen.git#77031560146c6985c99cd40eda208b04052ab631",
     "ts-loader": "^2.0.0",
-    "typescript": "^2.5.0",
+    "typescript": "^2.6.2",
```

Comment:
had to set 2.6 to be able to use ts-ignore for the global variables coming from the define plugin